wallet-new/server/Cardano/Wallet/API/V1/Swagger/Example.hs

```diff
@@ -48,7 +48,7 @@ instance Example WalletId
 instance Example BackupPhrase
 instance Example (V1 BackupPhrase)
 instance Example AssuranceLevel
-instance Example SyncProgress
+instance Example SyncPercentage
```

Comment:
I have renamed the `SyncProgress` to just `SyncPercentage`, as the former now is used to capture a much richer set of information about the syncing process.